test/e2e/setup.go

```diff
@@ -102,8 +125,12 @@ var _ = BeforeSuite(func() {
 	SetDefaultEventuallyPollingInterval(10 * time.Second)
 
 	for _, key := range []string{
+		"AZURE_CLIENT_ID",
```

Comment:
These are not mandatory in CI, but for local env context, they are. Not sure what is better here. Check `getAuthorizer` function for more details. 

The alternative is to set ENV vars in CI, which Im trying to avoid.